.zuul.playbooks/playbooks/wheel/roles/build-wheel-manylinux/files/build-wheels.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash -x
+# Compile wheels
+
+cd /io
+mkdir wheelhouse.final
+
+for PYBIN in /opt/python/*/bin; do
+
+    "${PYBIN}"/python -m virtualenv .venv
+
+    .venv/bin/pip install cffi six ipaddress "enum34; python_version < '3'"
+
+    REGEX="cp3([0-9])*"
+
+    #if [[ "${PYBIN}" =~ $REGEX ]]; then
+    #    PY_LIMITED_API="--build-option --py-limited-api=cp3${BASH_REMATCH[1]}"
+    #fi
+
+    LDFLAGS="-L/opt/pyca/cryptography/openssl/lib" \
+           CFLAGS="-I/opt/pyca/cryptography/openssl/include -Wl,--exclude-libs,ALL" \
+           .venv/bin/pip wheel /io --no-binary cryptography --no-deps --wheel-dir=tmpwheelhouse $PY_LIMITED_API
```

Comment:
`pip wheel` turns out to be a real pain due to PEP517 issues. We should instead download, untar, and run python setup.py bdist_wheel. We also want to do a trivial test on the built wheel (you can see what we normally do here: https://github.com/pyca/cryptography/blob/master/.github/workflows/wheel-builder.yml#L24-L49)

Reply:
> We also want to do a trivial test on the built wheel (you can see what we normally do here: https://github.com/pyca/cryptography/blob/master/.github/workflows/wheel-builder.yml#L24-L49)

One slight issue with replicating that is that aarch64 doesn't have prelink (which provides execstack) and it won't be available (https://bugzilla.redhat.com/show_bug.cgi?id=1460531)